src/main/java/uk/gov/laa/ccms/caab/bean/validators/client/ClientContactDetailsValidator.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.laa.ccms.caab.bean.validators.client;
+
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import org.springframework.validation.Errors;
+import uk.gov.laa.ccms.caab.bean.ClientDetails;
+import uk.gov.laa.ccms.caab.bean.validators.AbstractValidator;
+
+/**
+ * Validator component responsible for validating {@link uk.gov.laa.ccms.caab.bean.ClientDetails}
+ * objects.
+ */
+@Component
+public class ClientContactDetailsValidator extends AbstractValidator {
+
+  /**
+   * Determines if the Validator supports the provided class.
+   *
+   * @param clazz The class to check for support.
+   * @return {@code true} if the class is assignable from
+   *         {@link uk.gov.laa.ccms.caab.bean.ClientDetails}, {@code false} otherwise.
+   */
+  @Override
+  public boolean supports(Class<?> clazz) {
+    return ClientDetails.class.isAssignableFrom(clazz);
+  }
+
+  /**
+   * Validate the telephone numbers in the {@link ClientDetails}.
+   *
+   * @param clientDetails The object to be validated.
+   * @param errors The Errors object to store validation errors.
+   */
+  private void validateTelephones(ClientDetails clientDetails, Errors errors) {
+    // At least one telephone number should be provided
+    if (!StringUtils.hasText(clientDetails.getTelephoneHome())
+        && !StringUtils.hasText(clientDetails.getTelephoneHome())
```

Comment:
Copy paste error - guess it should be work and mobile?

Reply:
Amended copy paste error